tests/config.ini

```diff
@@ -0,0 +1,15 @@
+#Do NOT change the section name
```

Comment:
pasting a diff as this file is already checked in earlier.. don't know why github didn't just picked up the diff rather than displaying the whole content

diff --git a/tests/config.ini b/tests/config.ini
index aa0bb6d..d5a3bae 100644
--- a/tests/config.ini
+++ b/tests/config.ini
@@ -2,15 +2,14 @@

 # please provide the Nimble Array credentials against which the test case will be run. Note that the password is plain text and hence when the same is released to public github it will be removed
 [NimbleArrayCredentials] #section name
-hostname: 10.18.161.229
-#hostname: 10.18.171.96
-username: admin
-password: admin
+hostname: 1.1.1.1
+username: ******
+password: ******